packages/cysync-core-constants/src/types/serverCoinError/index.ts

```diff
@@ -0,0 +1,62 @@
+import { CoinFamily } from '@cypherock/coins';
+
+import { LangErrors } from '../../i18n/types';
+import { IErrorMsg } from '../deviceError';
+import { ServerErrorDetails } from '../serverError';
+
+// fill this variable as u define coin errors: ServerCoinErrorTypes = XrpErrorType | BtcErrorType | EvmErrorType
+export type ServerCoinErrorTypes = 'unknown';
+
+export type ServerCoinErrors = Partial<
+  Record<CoinFamily, Record<ServerCoinErrorTypes, IErrorMsg>>
```

Comment:
This should be the fallback text consisting of a `{message:string}` instead of `IErrorMsg`